OpenRA.Mods.Common/Traits/Radar/JamsRadar.cs

```diff
@@ -14,11 +14,24 @@
 namespace OpenRA.Mods.Common.Traits.Radar
 {
 	[Desc("When an actor with this trait is in range of an actor with ProvidesRadar, it will temporarily disable the radar minimap for the enemy player.")]
-	public class JamsRadarInfo : TraitInfo<JamsRadar>
+	public class JamsRadarInfo : ITraitInfo
 	{
 		[Desc("Range for jamming.")]
 		public readonly WDist Range = WDist.Zero;
+
+		[Desc("What diplomatic stances are affected.")]
+		public readonly Stance JamsRadarStances = Stance.Enemy | Stance.Neutral;
```

Comment:
We have `DeflectionStances` in `JamsMissiles`, so `JamStances` here for "consistency"?